test/modules/aws-ecs-integration.ts

```diff
@@ -378,6 +492,14 @@ describe('ECS Integration Testing', () => {
       WHERE service.name = '${serviceRepositoryName}';
     `, (res: any[]) => expect(res.length).toBe(1)));
 
+    it('applies service insertion', apply());
```

Comment:
we actually were never applying the service insertion here which increased the test time